src/doc/src/reference/specifying-dependencies.md

```diff
@@ -268,10 +333,7 @@ and specify its version in the dependencies line as well:
 hello_utils = { path = "hello_utils", version = "0.1.0" }
 ```
 
-> **Note**: Neither the `git` key nor the `path` key changes the meaning of the
-> `version` key: the `version` key always implies that the package is available
-> in a registry. `version`, `git`, and `path` keys are considered [separate
-> locations](#multiple-locations) for resolving the dependency.
+The use of `path` and `version` keys together is explained in the next section.
```

Comment:
I replace the Note with this line because the Note didn't explain it well enough and the section that does explain it well is only a few lines below.

Reply:
I am slightly worried that “the next section” might be moved to elsewhere over time.

```suggestion
The use of `path` and `version` keys together is explained in the [Multiple locations](#multiple-locations) section. We can just call it out here.
````